h/views/api.py

```diff
@@ -304,6 +292,29 @@ def _json_payload(request):
         raise PayloadError()
 
 
+def _call_search_lib(request, json_presenter=None):
```

Comment:
The `json_presenter` argument should not have a default value because the code in this function does not support a `None` value (`find_service(name=None)` will throw an exception).